cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/MapReduceWithPartitionedTest.java

```diff
@@ -169,9 +169,9 @@ public void apply() {
       new TransactionExecutor.Subroutine() {
         @Override
         public void apply() {
-          Partition partition = tpfs.getPartitionByTime(time);
-          Assert.assertNotNull(partition);
-          String path = partition.getRelativePath();
```

Comment:
this change is actually not needed, because you never call getMetadata(), it should be fine to retrieve it as a Partition